src/lib/components/workflow/workflow-json-navigator.svelte

```diff
@@ -78,12 +82,27 @@
     </div>
     <slot name="decode" />
   </div>
-  <CodeBlock
-    content={stringifyWithBigInt(events[index - 1])}
-    testId="event-history-json"
-    copyIconTitle={translate('copy-icon-title')}
-    copySuccessIconTitle={translate('copy-success-icon-title')}
-  />
+  {#if decodeEventHistory}
+    {#key [index, decodeEventHistory]}
```

Comment:
TIL you can do array keys. is the `decodeEventHistory` necessary so that we re-render encoded/decoded payloads when the switch is toggled on and off?

Reply:
Yup exactly